selinux/systemd-tmpfiles-setup-sys.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Set correct SELinux labels in /sys
+Documentation=man:tmpfiles.d(5) man:systemd-tmpfiles(8)
+
+DefaultDependencies=no
+Before=sysinit.target local-fs-pre.target
+Wants=local-fs-pre.target
+Conflicts=shutdown.target initrd-switch-root.target
+Before=shutdown.target initrd-switch-root.target
+ConditionSecurity=selinux
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=systemd-tmpfiles --prefix=/sys --create --boot
+SuccessExitStatus=DATAERR CANTCREAT
```

Comment:
The upstream unit also has `ImportCredential=tmpfiles.*`

Reply:
i am not 100% sure if we need this, do we want users to be able to add more lines to the systemd-tmpfiles in that service? they could do with the normal systemd-tmpfiles-setup.service

Reply:
Not sure what you mean by that

Reply:
readding after offline discussion